### PR TITLE
 Move guides to community section 

### PR DESCRIPTION
**Pull request by lukecd**
This pull request moves the Irys guides into the community section

### CODE REVIEW

**Comment by hadifalahi4142** [comment]
I think happen soon

File: sidebars.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const sidebars = {
       collapsed: true,
       label: "Move funds to Linea",
       link: {
```
Comment (emilyJLin95) [inline sidebars.js:9]: Was this all linter changes?

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]
https://github.com/Consensys/doc.linea/actions/runs/9272494953/job/25510411724?pr=575

You'll need to add words to `project-words.txt` and run `npm run lint` to make sure everything passes

**Comment by bgravenorst** [comment]
Closing in favour of https://github.com/Consensys/doc.linea/pull/594
Which duplicates this content and fixes cspell and includes edits for styling and readability.

**Pull request closed without merging**